### PR TITLE
User edit account

### PR DESCRIPTION
**Pull request by lilybarrett**
Feature: User edits account info

### CODE REVIEW

File: log/development.log [hunk 1]
```diff
@@ -2492,6 +2492,136 @@ Completed 200 OK in 37ms (Views: 35.9ms | ActiveRecord: 0.3ms)
 
 Started GET "/assets/application.self-e80e8f2318043e8af94dddc2adad5a4f09739a8ebb323b3ab31cd71d45fd9113.css?body=1" for ::1 at 2016-01-04 17:14:22 -0500
 
+<<<<<<< HEAD
```
Comment (alxjrvs) [inline log/development.log:2495]: You should remove your `log` directory from git - logs aren't really useful to save in version control.

File: spec/features/user_edits_account_spec.rb [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    fill_in 'Password Confirmation', with: 'newpassword'
+    fill_in 'Current Password', with: user.password
+    click_button 'Update'
+
+    expect(page).to have_content("Your account has been updated successfully.")
+    expect(page).to have_content('Sign Out')
+  end
+
+  scenario 'user does not provide required information' do
+    user = FactoryGirl.create(:user)
```
Comment (alxjrvs) [inline spec/features/user_edits_account_spec.rb]: Consider using a `before` block in these tests to dry them up a bit!

File: app/controllers/registrations_controller.rb [hunk 3]
```diff
@@ -5,9 +5,4 @@ class RegistrationsController < Devise::RegistrationsController
   #   flash[:notice] = "You're in!"
   #   :edit_user_registration
   # end
-  #
-  # def after_inactive_sign_up_path_for(_resource)
```
Comment (alxjrvs) [inline app/controllers/registrations_controller.rb:9]: Why did you delete this?

Comment (willxfeng) [inline app/controllers/registrations_controller.rb:9]: When a user signs up for an account, the default behavior is to redirect to root path, which did not exist when we first started coded this feature. That is why we added the above code to change the redirect path to a path that does exist (i.e. edit user registration). Now that we have merged changes made by the other pair in the group, we not have root path that redirects to the index page. So we no longer need these commented out lines

File: log/development.log (hunk 1 above)
Comment (lilybarrett) [inline log/development.log:2495]: It's odd -- We had added it to our .gitignore file, but .git somehow seems to still be tracking it. We plan to copy the contents of the file, delete, recreate, and paste...

File: spec/features/user_edits_account_spec.rb (hunk 2 above)
Comment (lilybarrett) [inline spec/features/user_edits_account_spec.rb]: Made these changes! Stay tuned.

File: log/development.log (hunk 1 above)
Comment (alxjrvs) [inline log/development.log:2495]: Ah! So, the file is currently being tracked by `git`, so adding it to the gitignore alone is not sufficient. 

Run `git rm -rf --cached log`. this should remove git's history of the file (hence the `--cached` - "only remove the cached version git has, not the real version I have on my computer") and, since you have it attached as a .gitignore, it shouldn't try to add it anymore.

File: app/controllers/registrations_controller.rb (hunk 3 above)
Comment (alxjrvs) [inline app/controllers/registrations_controller.rb:9]: Well explained!
